relValTools.py

```diff
@@ -88,10 +91,15 @@ def getFilesFromEOS(path, cmseospath=True):
 
 def getFilesFromDAS(release, runtype, globalTag):
     '''Get proxy with "voms-proxy-init -voms cms" to use this option.'''
-    query = "file dataset=/*{0}*/*{1}*{2}*/MINIAODSIM".format(runtype, release, globalTag, )
-    print "Getting files from DAS. May take a while.... query:", query
-    result = subprocess.check_output("dasgoclient --query='" + "file dataset=/*{0}*/*{1}*{2}*/MINIAODSIM".format(runtype, release, globalTag, ) + "'", shell=True)
+    query = "file dataset=/{0}/{1}-{2}/MINIAODSIM".format(runtype, release, globalTag, )
+    print "Getting files from DAS. query:", query
+    result = subprocess.check_output("dasgoclient --query='" + query + "'", shell=True)
     files = ["root://cms-xrd-global.cern.ch/" + s.strip() for s in result.splitlines()]
+    if files == []:
```

Comment:
You can shorten this a bit if you check 
    if result:
before constructing the files list, then re-query, and have only one of the
    files = ...
statements.

Reply:
Ok, I fixed it.